src/components/FilterBar.jsx

```diff
@@ -19,7 +19,13 @@ export default function FilterBar({ setQueryParam, queryParams }) {
       <Grid item>
         <MajorFilter
           majors={majors}
-          changeFn={(e) => setQueryParam(qParams.MAJORS, e)}
+          onSelect={(e) => setQueryParam(qParams.MAJORS, e)}
+          onDelete={(e) =>
```

Comment:
`e` is normally a shorthand for `event` in listeners and the value is normally `e.target.value`. How about using `m` instead?